src/main.js

```diff
@@ -191,3 +191,4 @@ const draw = () => {
 addColours()
 
 form.addEventListener("input", debounce(draw, 200));
+customPositionInput.addEventListener("change", debounce(draw, 200));
```

Comment:
Is this needed? The form `input` event handler should be enough here, if the slider is part of that form.

Reply:
It is needed because the form is only submitted once the slider is released.  To get dynamic positioning without releasing the mouse, the change event must be consumed.

Reply:
Could we add a comment for future travellers? A possible alternative solution could be to listen to the form's `change` event rather than `input`.